Client/Frontend/Home/FxHomeRecentlySavedCollectionCell.swift

```diff
@@ -76,12 +80,14 @@ class FxHomeRecentlySavedCollectionCell: UICollectionViewCell {
         ])
     }
     
-    private func loadItems() -> [RecentlySavedItem] {
+    public func loadItems() -> [RecentlySavedItem] {
         var items = [RecentlySavedItem]()
         
         items.append(contentsOf: recentBookmarks)
         items.append(contentsOf: readingListItems)
         
+        Self.recentlySavedItems = items
```

Comment:
nit: self is capitalized

```suggestion
        self.recentlySavedItems = items
```

You can ignore this if you don't plan on using static vars